source/css/pattern-scaffolding-project-specific.scss

```diff
@@ -87,7 +87,7 @@ pl-iframe {
 
 .pl-c-category__title,
 .pl-c-pattern__title {
-	@extend .elm-headline;
+	//@extend .elm-headline;
 }
```

Comment:
if we don't need a declaration any more, let's just remove it on total. Due to the versioning system we could still understand any changes, but would make our changes more clearer. In case that you're just "commenting out" this code piece and would need some rework or have questions, please leave a `TODO` comment

Reply:
removed headline element from core